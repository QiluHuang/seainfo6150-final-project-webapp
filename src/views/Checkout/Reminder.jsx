import React from 'react';
import styles from './Reminder.module.css';
import PropTypes from "prop-types";


const Reminder = (props) => {
    return (
        <div className={styles.container}>
            <div className={styles.image}><img src="https://cdn11.bigcommerce.com/s-7j8u58/images/stencil/1110x1110/uploaded_images/miami-healthy-meals-delivery-service-the-best-way-to-time-your-meals-for-weight-loss.jpg?t=1561061586" alt={"background"}/></div>

            <div className={styles.background}>
                <div className={styles.text}>Please get your meal at {props.text}</div>
            </div>
        </div>
    );
}

Reminder.propTypes = {
    text: PropTypes.string.isRequired,
};

export default Reminder;