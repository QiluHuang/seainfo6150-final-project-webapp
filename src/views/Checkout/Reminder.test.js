import React from "react";
import { render } from "@testing-library/react";
import Reminder from "./Reminder.jsx";

describe("Reminder tests", () => {
    it("renders the Reminder component correctly", () => {
    const { container } = render(<Reminder text={"10:30PM"} />);
    expect(container).toMatchSnapshot();
    });
});