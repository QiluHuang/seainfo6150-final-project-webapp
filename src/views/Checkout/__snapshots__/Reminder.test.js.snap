// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reminder tests renders the Reminder component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="image"
    >
      <img
        alt="background"
        src="https://cdn11.bigcommerce.com/s-7j8u58/images/stencil/1110x1110/uploaded_images/miami-healthy-meals-delivery-service-the-best-way-to-time-your-meals-for-weight-loss.jpg?t=1561061586"
      />
    </div>
    <div
      class="background"
    >
      <div
        class="text"
      >
        Please get your meal at 
        10:30PM
      </div>
    </div>
  </div>
</div>
`;
